src/pages/Documentation.js

```diff
@@ -1,56 +1,135 @@
-        </li>
-      </ul>
-      <h3>Queries</h3>
-      <p>
-        aliases, otherRelations, affiliation, occupation, residence,
-        appearsInEpisodes, status, gender, eyeColor, born, hairColor,
-        portrayedBy
-      </p>
+      <ExpansionPanel
+        classes="endpoints-list"
+        style={panelStyle}
+      >
+        <ExpansionPanelSummary
+          expandIcon={expandIcon}
+          aria-controls="panel1a-content"
+          id="panel1a-header"
+          classes="endpoints-list"
+        >
+          <span className="bold">GET all characters</span> - /api/v1/characters
+        </ExpansionPanelSummary>
+        <ExpansionPanelDetails
+          className="example-json"
+        >
+          <pre>
+            {JSON.stringify(data.all, null, 1)}
+          </pre>
+        </ExpansionPanelDetails>
+      </ExpansionPanel>
+      <ExpansionPanel
+        classes="endpoints-list"
+        style={panelStyle}
+      >
+        <ExpansionPanelSummary
+          classes="endpoints-list"
+          expandIcon={expandIcon}
+          aria-controls="panel1a-content"
+          id="panel1a-header"
+        >
+          <span className="bold">GET by ID</span> - /api/v1/characters/:id
+        </ExpansionPanelSummary>
+        <ExpansionPanelDetails
+          className="example-json"
+        >
+          <pre>
+            {JSON.stringify(data.id, null, 1)}
+          </pre>
+        </ExpansionPanelDetails>
+      </ExpansionPanel>
+      <ExpansionPanel
+        style={panelStyle}
+      >
+        <ExpansionPanelSummary
+          classes="endpoints-list"
+          expandIcon={expandIcon}
+          aria-controls="panel1a-content"
+          id="panel1a-header"
+        >
+          <span className="bold">GET random character(s)</span> - /api/v1/characters/random?count=1
+        </ExpansionPanelSummary>
+        <ExpansionPanelDetails
+          className="example-json"
+        >
+          <pre>
+            {JSON.stringify(data.random, null, 1)}
+          </pre>
+        </ExpansionPanelDetails>
+      </ExpansionPanel>
+      <ExpansionPanel
+        style={panelStyle}
+      >
+        <ExpansionPanelSummary
+            classes="endpoints-list"
+            expandIcon={expandIcon}
+            aria-controls="panel1a-content"
+            id="panel1a-header"
+          >
+            <span className="bold">GET by query</span> - /api/v1/characters?name=Mike+Wheeler
+          </ExpansionPanelSummary>
+          <ExpansionPanelDetails
+            className="example-json"
+          >
+            <div>
+              <h3>Available Queries</h3>
+              <p>
+                aliases, otherRelations, affiliation, occupation, residence,
+                appearsInEpisodes, status, gender, eyeColor, born, hairColor,
+                portrayedBy
+              </p>
+            </div>
+            <pre>
+              {JSON.stringify(data.name, null, 1)}
+            </pre>
```

Comment:
One thing I noticed on this one is that I thought the scroll wasn't working at first because I had my cursor on top of the queries instead of the code. Could we add some kind of color difference or border or something to show that one section is scrollable and the other isn't?

Reply:
Ooh good call. I like the idea of a border. What do you think about pink? And should that exist on the code blocks on the other expansionPanels to be a signal of "scrollable content"?

Reply:
Yeah! Sounds good to me! 💅